utils/imageUtils.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * pathUtils.js
+ * A common set of utility functions related to uploading / importing
+ * files.
+ */
+const { exec } = require("child_process");
+const fs = require("fs");
+const { reject } = require("lodash");
+const path = require("path");
+
+module.exports = {
+   convert: async (imagePath, extension, quality, resize) => {
+      switch (extension) {
```

Comment:
suggest:
```
const validExtensions = ["heic", "jpg", ...];
if (!validExtendions.includes(extension)) return;
```

Reply:
I like this.  
Also, if we could keep this someplace more central like: `AB.objectFile().validExtensions`  so we can keep 1 copy of this info and use it multiple places.

Reply:
https://github.com/digi-serve/appbuilder_platform_service/pull/90